components/annot-tab.vue

```diff
@@ -161,7 +161,8 @@
             this.category = this.details.category;
 
             // 2. Get column marked as subject ID
-            this.idField = this.getColumnOfCategory("Subject ID");
+            // NOTE: Subject ID is only allowed one column
```

Comment:
Maybe this would be clearer here:
```suggestion
            // NOTE: the "Subject ID" category is only allowed to be applied to one column
```